odysseus/convertFtbQuests.ts

```diff
@@ -134,14 +138,16 @@ type QuestTask = QuestObject & ({
 
 type QuestReward = BasicQuestObject & (Advancement | {
     type: FtbId<'choice'>;
+    table_id: string;
```

Comment:
Should be Long.

Reply:
I would love to have the answer to that question:

![image](https://github.com/terrarium-earth/odysseus/assets/55819817/b4bb3f92-d948-4357-814c-227c646acfb4)

![image](https://github.com/terrarium-earth/odysseus/assets/55819817/059b06c3-5444-4abc-bba9-3b1cc8146f92)

![image](https://github.com/terrarium-earth/odysseus/assets/55819817/edac0273-a509-4547-a058-e048931ef74b)

in practice, I know I had to manually replace stringified counts, amounts, and lvls with a pure number in json, so i'm playing defence.

Reply:
Yep long, so use the defined `Long` type. (it's defined as string but just so it's clearer)

Reply:
(For clarification)
https://github.com/terrarium-earth/odysseus/blob/main/odysseus/convertFtbQuests.ts#L21

Reply:
done in 644b643

Reply:
Oh sorry, my comment was meant for the other review message, where you sent a few images showing a long count